packages/esm-patient-search-app/src/hooks/usePatients.tsx

```diff
@@ -0,0 +1,37 @@
+import { useMemo } from 'react';
+import { fhirBaseUrl, openmrsFetch } from '@openmrs/esm-framework';
+import useSWR from 'swr';
+
+interface FHIRResponse {
+  resourceType: string;
+  id: string;
+  meta: object;
+  type: string;
+  total: number;
+  entry: Array<{ resource: fhir.Patient; fullUrl: string }>;
+}
+
+interface PatientReturnType {
+  patients: Array<fhir.Patient>;
+  isLoading: boolean;
+  error: any;
+}
+/**
+ * React hook that performs patient search using FHIR endpoint
+ * @param query The search term which can be identifier or name
+ * @param includeDead Whether to include dead patients in search results
+ * @returns Object {patients,isLoading,error}
+ */
+const usePatients = (query: string, includeDead: boolean = false): PatientReturnType => {
```

Comment:
Please change this to `usePatientSearch`

Reply:
@ZacButko have fixed the proposed changes. Thanks for the review.